src/bootlint.js

```diff
@@ -964,6 +966,86 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
+        var BOOTSTRAP_V4_ERROR = "Bootstrap version 4 detected. Bootlint currently supports Bootstrap version " + CURRENT_BOOTSTRAP_VERSION;
```

Comment:
Since this is a heuristic, how about "Detected what appears to be Bootstrap v4." for the first part?